### DISCUSSION TITLE
distinction between diseases and phenotypes

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#2598] [Mondo design decisions]
# This is an example use case

The sub-classification of MONDO_0010850 Tessier number 4 facial cleft as a subclass of MONDO_0015824 Oculomaxillofacial dysostosis seems incorrect (although I can't identify the cause - the xref of [orphanet_141258](http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=GB&Expert=141258) to [OMIM_600251](http://www.omim.org/entry/600251?search=Tessier%20number%204%20facial%20cleft&highlight=cleft%20tessier%20number%20facial%204) seems okay.

What action should I take here? Should I just remove the parent class assertion in Protege and make a pull request?

Thanks!

![image](https://user-images.githubusercontent.com/6722114/32866093-ce04eb3c-ca1a-11e7-9331-1fc66699ed39.png)

**Comment by kanems** [comment]
Is this term equivalent to https://hpo.jax.org/app/browse/term/HP:0031576 ? Based on the names and cross-references, I'm not sure how to map this concept for MedGen. 

UMLS has different identifiers for the HPO term and the OMIM presentation, and the names/synonyms used by the various groups seems to be variably interchangeable...
 The OMIM number is on UMLS C1838348, "oculomaxillofacial dysostosis" which is the name on the parent term in MONDO->MONDO:0015824.  OMIM treats "tessier type 4" as a phenotype (in line with HPO) and describes 4 affected patients from a Richieri-Costa and Gorlin 1994 paper under "clinical variability" on the "OCULOMAXILLOFACIAL DYSPLASIA WITH OBLIQUE FACIAL CLEFTS" entry (600251)

For additional X-refs: GARD 4046 treats Oculomaxillofacial dystosis as equivalent to : " Richieri Costa Gorlin syndrome; Oculomaxillofacial dysplasia with oblique facial clefts; Oblique facial clefts" https://rarediseases.info.nih.gov/diseases/4046/index

So perhaps the name "Tessier number 4..." should not be the preferred name on this term, at a minimum, to clarify the clinical presentation as a whole is not equivalent to the precise presentation in the HPO term?

@pnrobinson @ahamosh @ana-rath-orphanet

**Comment by pnrobinson** [comment]
This again gets to the distinction between diseases and phenotypes. If a person has a tessier type 4 cleft as their only clinical problem, it is possible to treat it as a disease. On the other had, various kinds of clefts can be observed as features of other diseases. 
In general, I would propose that a disease consists of a clincial course, response to treatment, and an etiology. This distinguishes it from a phenotypic feature. So, a good MONDO term might be "Idiopathic Tessier type 4 cleft" or "Isolated Tessier type 4 cleft". The corresponding HPO terms might include Tessier number 4 facial cleft HP:0031576 and perhaps terms related to common associated findings such as difficulty with feedings, difficulty swallowing, nasal voice and recurrent otitis media.
@cmungall @mellybelly @ahamosh I think it is really important not to make ad hoc decisions about these issues but to have a comprehensive written statement about MONDO's philosophy. We really need to have some workshops!

**Comment by nicolevasilevsky** [comment]
see also: https://github.com/monarch-initiative/mondo/issues/86

**Comment by nicolevasilevsky** [comment]
see also https://github.com/monarch-initiative/mondo/issues/559

**Comment by nicolevasilevsky** [comment]
another example: https://github.com/monarch-initiative/mondo/issues/501

**Comment by ahamosh** [comment]
These are all excellent examples of the problem with phenotypic feature (which belongs in HPO) and a diagnosis (which belongs in Mondo).  The problem becomes even more complicated when a diagnosis is a feature of a disease (e.g. diabetes as part of MELAS (mitochondrial encephalopathy lactic acidosis and stroke).  The Discitis example is weird:  it is an inflammation of the disc which could be a feature or a diagnosis, especially a diagnosis if the etiology is infection.  I think that all of this is/will be very hard.  The Tessier type 4 cleft is the description of the cleft.  It is definitely a feature and probably of the syndrome, but it sounds like in some people the cleft is the extent of the syndrome...

**Comment by ana-rath-orphanet** [comment]
Hello,
We are trying to add "isolated" to a morphological abnormality when it is the disorder (not complete yet), and all the disorder. We then attribute a classification_level=disorder and a typology:morphological anomaly. Having a disease code (i.e. Orpha) for those should help distinguishing them from phenotypic abnormalities that have an HPO code. Plenty of examples: you can download and list them by filtering by morphologuical anomaly out of the Orphanet nomenclature @annieolry

**Reply by ahamosh** [reply]
That is a good idea, but we will have a lot of overlap.

**Reply by pnrobinson** [reply]
I agree this is a good idea -- I do not think it makes more overlap, instead it should help to disambiguate the existing overlap.

**Comment by nicolevasilevsky** [comment]
https://github.com/monarch-initiative/mondo/issues/822
